src/Components/PresentationalComponents/Filters/PrimaryToolbarFilters/SecurityRuleFilter.js

```diff
@@ -7,26 +7,34 @@ import unionWith from 'lodash/unionWith';
 import isEqual from 'lodash/isEqual';
 
 const securityRuleFilter = (apply, currentFilter = {}, dynamicFilters = []) => {
-    let { security_rule: currentValue } = currentFilter;
-    if (currentValue === '' || !currentValue) {
-        currentValue = 'all';
+    let currentValue = [];
+    if (currentFilter.rule_presence) {
+        currentValue = currentValue.concat(currentFilter.rule_presence.split(','));
+    }
+
+    if (currentFilter.rule_key) {
+        currentValue = currentValue.concat(currentFilter.rule_key.split(','));
     }
 
     const filterBySecurityRule = values => {
-        apply({ security_rule: values === 'all' ? '' : values, page: 1 });
+        apply({
+            rule_presence: values.filter(value => value === 'true' || value === 'false').join(',') || undefined,
```

Comment:
remember the principle don't repeat yourself (DRY) 
The following block is can be a method so it can be re-used
`values.filter(value => value === 'true' || value === 'false').join(',') || undefined`

Reply:
But the conditions are opposite, I will change it to be more clear:
`['true', 'false'].includes(value)`
and
`!['true', 'false'].includes(value)`